lib/routes/account.js

```diff
@@ -853,6 +858,12 @@ module.exports = (log, db, mailer, Password, config, customs, signinUtils, push)
             }
           })
           .then(() => {
+            // If no keys set on the response, there was no valid profile scope found. We only
+            // want to return `profileChangedAt` if a valid scope was found and set.
+            if (Object.keys(res).length !== 0) {
```

Comment:
I wasn't sure if this should only be included on a certain scope, and if so which scope to include.

Reply:
The logic you have here seems right to me - return `profileChangedAt` if and only if we're returning a non-empty object with some profile info.